e2etest/GuestProxyAgentTest/LinuxScripts/GuestProxyAgentExtensionValidation.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+customOutputJsonUrl=$(echo $customOutputJsonSAS | base64 -d)
+expectedProxyAgentVersion=$(echo $ExpectProxyAgentVerison)
+echo "extractedVersion=$expectedProxyAgentVersion"
```

Comment:
log as expectProxyAgentVerison=$expectProxyAgentVerison

Reply:
why you resolve it without any change? I still see you log as "extracedVersion="